### PR TITLE
Dev/doc updates

### PR DESCRIPTION
**Pull request by abierbaum**
Misc updates as I go through and try to get everything running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,3 +101,7 @@ TODO
  * extend doc tool to show `@private` flag
 
  * Add styling
+
+References
+==========
+ * The project layout and structure is based upon https://github.com/joshdmiller/ng-boilerplate . See that project for more description on the *why* of the layout.
```
Comment (my8bird) [inline README.md]: I try to wrap all lines so that the diff does not get scrollers.  right after the link would work great

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {},
   "devDependencies": {
     "grunt": "~0.4.1",
+    "grunt-cli": "~0.1.9",
```
Comment (my8bird) [inline package.json:14]: yeah, I think i installed this one manually early on

File: README.md [hunk 3]
```diff
@@ -22,6 +24,7 @@ Run It
 ------
 ```bash
 # Open the index page in your browser
+# TODO: This doesn't work anymore...
 chromium-browser build/index.hmtl
```
Comment (my8bird) [inline README.md:28]: I think we can just change this to `Open your browser to http://localhost:9000/`

Comment (abierbaum) [inline README.md:28]: Depends upon what the default task is I guess.  Right now, it wouldn't work.

File: README.md (hunk 1 above)
Comment (abierbaum) [inline README.md]: Done.

**Pull request merged**